CMakeLists.txt

```diff
@@ -84,6 +84,7 @@ include(var_functions)
 option(NGRAPH_UNIT_TEST_ENABLE "Control the building of unit tests" TRUE)
 option(NGRAPH_TOOLS_ENABLE "Control the building of tool" TRUE)
 option(NGRAPH_CPU_ENABLE "Control the building of the CPU backend" TRUE)
+option(NGRAPH_INTELGPU_ENABLE "Control the building of the CPU backend with clDNN" TRUE)
```

Comment:
I implement it as TRUE to involve the new code into regular build procedure for all other development work (in all incoming PRs). If we keep it switched FALSE it will be potentially broken by other development work due no regular build tests.
Anyway, I will keep in FALSE as you like.